docs/src/ci.md

```diff
@@ -312,7 +312,7 @@ tests:
 
 By default, Playwright downloads browser binaries when the Playwright NPM package
 is installed. The NPM packages have a `postinstall` hook that downloads the browser
-binaries. This behavior can be [customized with environment variables][installation-managing-browser-binaries].
```

Comment:
please follow up on playwright.dev and remove this weird link which I've created in the params when rolling.

Reply:
Yes, I started there.